hub/management/commands/import_council_data.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss-code"
+    message = "Importing council population data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "data_type": "integer",
+        "category": "place",
+        "subcategory": "",
+        "release_date": "February 2023",
+        "label": "Council population",
+        "source_label": "Data from mySociety.",
+        "source": "https://www.mysociety.org/",
+        "source_type": "csv",
+        "table": "areadata",
+        "default_value": 1000,
```

Comment:
Don’t we typically set this to something close to the average or the mode?

Reply:
we do use the average which is calculated later in the process. This is just a placeholder to fall back on.